test/base-warnings.js

```diff
@@ -0,0 +1,11 @@
+var tape = require('tape');
+var mjAPI = require("../lib/main.js");
+mjAPI.config({undefinedCharError: true});
+
+tape('basic test: check warnings', function (t) {
+    t.plan(2);
+    mjAPI.typeset({math:'呵', html:true})
```

Comment:
I'd prefer to see the math character as a unicode reference (`'\u5475`) rather than rely on the proper encoding of the javascript file.